apps/slackbot/task_agent.py

```diff
@@ -119,6 +121,14 @@ def run(self, task: str) -> str:
                 user_input=user_input,
             )
             print("reply:", assistant_reply)
+            # added by JF
+            try:
+                reply_json = json.loads(assistant_reply)
+                logger_step['reply'] = reply_json
+            except:
```

Comment:
Using a generic except is not a very good practice. After some search I found the `json.loads` function thoughts [`JSONDecodeError`](https://docs.python.org/3/library/json.html#json.JSONDecodeError) if the input is not a valid JSON. So we can catch that specific error instead like

``` python
try:
    json.loads(...)
except json.JSONDecoderError as e:
    ...
```